nsv13/code/modules/mining/asteroid.dm

```diff
@@ -160,6 +160,11 @@ GLOBAL_LIST_EMPTY(asteroid_spawn_markers)		//handles mining asteroids, kind of s
 	var/turf/target_location = null //Where to load the asteroid
 	var/cooldown = FALSE
 	var/tier = 1 //Upgrade via science
+	var/turf_type = /turf/open/space/basic
+
+//Hoc modo operatur. Do not question.
+/obj/machinery/computer/ship/mineral_magnet/stupidfuckingbabyaetherwhispmagnetvariantfortoproofdeckbecauseFUCKYOU
```

Comment:
A better rework of this is now required.

Reply:
Out of scope for this PR

Reply:
Agreed this is a terrible fix but it works for now